labels/mapping.go

```diff
@@ -107,4 +106,8 @@ var migrateMap = map[string]string{
 	"Enhancement":     "Type/Enhancement",
 	"Question":        "Type/Question",
 	"Tech/JavaScript": "",
+	// Migrate QA labels: https://community.mattermost.com/core/pl/eegcso8xr3bqzr7giftc3dgdka
+	"2: QA Review":        "3: QA Review",
+	"3: Reviews Complete": "4: Reviews Complete",
+	"QA Review Done":      "",
```

Comment:
I'd propose we leave this one untouched for now, and let QA iterate on it separately.

Reply:
Thanks.